### PR TITLE
Pull Request to Merge Docker Branch to Dev Branch

### PR DESCRIPTION
**Pull request by XPERIA679**
Merge Added Docker Environment Files to Dev Branch

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+MAIL_FROM_ADDRESS="hello@example.com"
+MAIL_FROM_NAME="${APP_NAME}"
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=us-east-1
+AWS_BUCKET=
+AWS_USE_PATH_STYLE_ENDPOINT=false
+
+VITE_APP_NAME="${APP_NAME}"
```
Comment (adPalafox) [inline .env:59]: Maybe not include this in the repo? I think it should be part of gitignore? 

Napansin ko lang 🙇

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  #   networks:
+  #     - microblog-network
+  #   command: sh -c "npm install && npm run dev"
+
+networks:
+  microblog-network: 
+    driver: bridge
+
+volumes:
+  mysql-data: {}
```
Comment (sunkenquest) [inline docker-compose.yml:68]: add new line, importante to lalo na kapag may conflicts sa file para alam niyo kung saan agad magsstart. Please apply it to all files.

File: docker/node/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+RUN npm install
+
+# Copy the rest of the application code
+COPY . .
+
+# Expose any ports the app is expecting
+EXPOSE 3000
+
+# Start the app
+CMD ["npm", "start"]
```
Comment (mikco-hipe) [inline docker/node/Dockerfile:19]: add new line here

File: makefile [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+laravel-project:
+	docker-compose exec php composer create-project laravel/laravel /var/www/html/
+
+migrate:
+	docker-compose exec php php /var/www/html/artisan migrate
+
+down-up:
+	docker-compose down && docker-compose up -d --build
+
```
Comment (mikco-hipe) [inline makefile:22]: add new line here

**Comment by sunkenquest** [comment]
Regarding the gitignore, use this reference in order to not include the .env file every time you commit changes
https://stackoverflow.com/questions/30696930/how-to-hide-env-file-from-github

**Review by sunkenquest — DISMISSED** [review]
LGTM 💯

**Review by sunkenquest — APPROVED** [review]
LGTM 💯

**Pull request merged**